public/fr/.htaccess

```diff
@@ -1,8 +1,8 @@
 <ifModule mod_rewrite.c>
-    RewriteEngine On
-    RewriteCond %{REQUEST_FILENAME} !-f
-    RewriteCond %{REQUEST_FILENAME} !-d
-    RewriteRule (.*) index.html [QSA,L]
```

Comment:
@awong-dev this seems to work for me locally for everything except the site.manifest